### PR TITLE
[harbour-validator] Allow Sailfish Secrets. JB#58500

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: allowed_requires.conf [hunk 1]
```diff
@@ -35,6 +35,10 @@ sailfish-components-webview-qt5-popups
 sailfish-components-webview-qt5-pickers
 libsailfishwebengine.so.1(*)
 
+# ### Sailfish Secrets framework
+libsailfishcrypto.so.0(*)
```
Comment (rainemak) [inline allowed_requires.conf:39]:
```suggestion
libsailfishsecretscrypto.so.0(*)
```

File: allowed_libraries.conf [hunk 2]
```diff
@@ -15,6 +15,10 @@ libsailfishapp.so.1
 libmdeclarativecache5.so.0
 libsailfishsilica.so.1
 
+# ### Sailfish Secrets framework
+libsailfishcrypto.so.0
```
Comment (rainemak) [inline allowed_libraries.conf:19]:
```suggestion
libsailfishsecretscrypto.so
```
Afais this should like this.

Comment (vigejolla) [reply allowed_libraries.conf:19]: Why? The library is libsailfishcrypto.so.0, there is no secrets in the name

Comment (rainemak) [reply allowed_libraries.conf:19]: It's actually both...
```
rpm -ql libsailfishsecretscrypto
/usr/lib64/libsailfishsecretscrypto.so.0
/usr/lib64/libsailfishsecretscrypto.so.0.2
/usr/lib64/libsailfishsecretscrypto.so.0.2.32
```

Comment (vigejolla) [reply allowed_libraries.conf:19]: Added libsailfishsecretscrypto. Also added the QML APIs and the Secrets permission.

**Review by rainemak — APPROVED** [review]

**Pull request merged**